src/ontology/mondo-edit.obo

```diff
@@ -251768,13 +251768,28 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0011174
 name: hyperzincemia with functional zinc depletion
+subset: gard_rare {source="GARD:17213", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disorder {source="Orphanet:251523"}
+subset: orphanet_rare {source="Orphanet:251523"}
+subset: otar {source="MONDO:OTAR"}
+subset: rare
+synonym: "hyperzincemia and hypercalprotectinemia" EXACT [Orphanet:251523]
 synonym: "hyperzincemia with functional zinc depletion" EXACT [OMIM:601979]
+synonym: "recurrent infections-inflammatory syndrome due to zinc metabolism disorder syndrome" EXACT [MONDO:0016676]
+xref: GARD:17213 {source="MONDO:GARD"}
+xref: ICD10CM:E83.2 {source="Orphanet:251523/attributed", source="Orphanet:251523/ntbt", source="Orphanet:251523"}
+xref: MEDGEN:1813911 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MEDGEN:356415 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:C566595 {source="MONDO:equivalentTo"}
 xref: OMIM:601979 {source="MONDO:equivalentTo"}
+xref: Orphanet:251523 {source="MONDO:equivalentTo"}
 xref: UMLS:C1865986 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:356415"}
+xref: UMLS:C4760957 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1813911"}
```

Comment:
and one of these UMLS xrefs should be MONDO:preferredExternal, too, right?

Reply:
These will be updated automatically by MedGen which maintains the MedGen x-refs.